readme.md

```diff
@@ -23,7 +23,7 @@ Anwendung in Spielen, Navigation, Planung, smarten Assistenten, autonomen Fahrze
 
 1.  Problemlösen
     *   Zustände, Aktionen, Problemraum
-    *   Suche (blind, informiert): Breiten-, Tiefensuche, Best-First,
+    *   Suche (blind, informiert): Breiten-, Tiefensuche, Best-First,i
```

Comment:
@cyildiz Wolltest Du hier noch was schreiben oder ist das "i" ein Versehen?

Reply:
Weiss nicht, wie das i hierher kam :)

Reply:
🤣 Du bist nicht zufällig mit VIM oder neoVim unterwegs? 

Dann mache ich das einfach flink raus ...

Reply:
Eigentlich nicht. NeoVim hört sich gut an, das schaue ich mir mal an. Danke für den Hinweis :)

Reply:
Hmmm, jaaaa. Es ist eine Erfahrung wert :) ... 

Ich kam darauf, weil im Vim zum Start des Edit-Modes einmal "i" gedrückt werden muss. Wenn ich ne Weile mit Vim unterwegs war und danach wieder mit "normalen" Editoren arbeite, finden sich in meinen Dokumenten anfangs einige zusätzliche "i" (und ":wq" zum Schreiben), weil das noch im Muskelgedächtnis ist und halt durchkommt, wenn man sich nicht konzentriert.